modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/css/EditApp.scss

```diff
@@ -269,6 +270,13 @@ $navbarHeight: 120px;
 	}
 }
 
+.dropdown-button-asset,
+.step-card-icon {
+	&.help-cursor {
+		position: absolute;
+	}
+}
+
```

Comment:
@gabrielpramos you can use the not operator `:not(.class)` to apply the `position: absolute` style only for the specific class.

For instance:

```scss

.dropdown-button-asset,
.icon-with-popover,
.step-card-icon {
	&.help-cursor {
	  //common style
	  
	  &:not(.icon-with-popover) {
	          position: absolute;
	  }
	}
```

You could try use this.

Reply:
Sure, thanks for reminding. I was thinking I was forgetting something 😊😊